docs/getting-started-guides/fedora/fedora_manual_config.md

```diff
@@ -20,37 +20,38 @@ The Kubernetes package provides a few services: kube-apiserver, kube-scheduler,
 
 Hosts:
 
-```conf
+```conf
 fed-master = 192.168.121.9
 fed-node = 192.168.121.65
-```
-
+```
+
 **Prepare the hosts:**
 
 * Install Kubernetes on all hosts - fed-{master,node}.  This will also pull in docker. Also install etcd on fed-master.  This guide has been tested with kubernetes-0.18 and beyond.
 * The [--enablerepo=updates-testing](https://fedoraproject.org/wiki/QA:Updates_Testing) directive in the yum command below will ensure that the most recent Kubernetes version that is scheduled for pre-release will be installed. This should be a more recent version than the Fedora "stable" release for Kubernetes that you would get without adding the directive.
 * If you want the very latest Kubernetes release [you can download and yum install the RPM directly from Fedora Koji](http://koji.fedoraproject.org/koji/packageinfo?packageID=19202) instead of using the yum install command below.
+* Running on AWS EC2 with RHEL 7.2, you need to enable extras repository for yum by editing /etc/yum.repos.d/redhat-rhui.repo and changing the changing the enable=0 to enable=1 for extras.

```

Comment:
* If running on AWS EC2 with RHEL 7.2, enable the "extras" repository for yum by editing `/etc/yum.repos.d/redhat-rhui.repo` and changing `enable=0` to `enable=1`.